pkg/ddc/alluxio/depericated_label.go

```diff
@@ -39,6 +40,11 @@ func (e *AlluxioEngine) HasDeprecatedCommonLabelname() (deprecated bool, err err
 
 	workers, err := e.getDaemonset(workerName, namespace)
 	if err != nil {
+		if apierrs.IsNotFound(err) {
+			deprecated = false
```

Comment:
Maybe we need indicate the labels are not found due to the daemonset is gone.

Reply:
Thanks for the advice. Added a log message for that.